app/src/main/contrib/components/Responsibilities/BurdenEstimates/UploadBurdenEstimatesContent.tsx

```diff
@@ -91,6 +89,6 @@ export const mapStateToProps = (state: ContribAppState): Partial<UploadBurdenEst
 };
 
 export const UploadBurdenEstimatesContent = compose(
-    connect(mapStateToProps, ),
+    connect(mapStateToProps, (dispatch, props) => props),
```

Comment:
Otherwise it adds a `dispatch` function to the props, which we don't need and it muddies the tests.